doio.c

```diff
@@ -2205,11 +2205,25 @@ Perl_do_print(pTHX_ SV *sv, PerlIO *fp)
         return TRUE;
     if (SvTYPE(sv) == SVt_IV && SvIOK(sv)) {
         assert(!SvGMAGICAL(sv));
-        if (SvIsUV(sv))
-            PerlIO_printf(fp, "%" UVuf, (UV)SvUVX(sv));
-        else
-            PerlIO_printf(fp, "%" IVdf, (IV)SvIVX(sv));
-        return !PerlIO_error(fp);
+        bool happy = TRUE;
```

Comment:
Why is `happy` declared so far away from its first use?

Reply:
Mirroring the `else` branch of `Perl_do_print`. I kept it it the same for consistency. Happy to change both instances in a follow-up commit.

Reply:
Yes, change